### ISSUE TITLE
increasing clarity of meaning (button>button groups and the grid)

### ISSUE THREAD

**Issue by FizbanUXD** [carbon-design-system/carbon-website#2607]
## Summary

TLDR: Buttons page>Button groups and the grid>into paragraph: **emphasize "both of which are acceptable"**

Long form narrative:
I was scrolling through the Buttons page and passed through the "Button groups and the grid" section. I was seeing the "do and don't" green and red tagged images, so my mental model was established for the page (and the website at large).

I liked the quickness of absorbing information visually like that, so I was stoked. But with this set of two images, I didn't know which was acceptable, and which was not. They's both acceptable, which is awesome, but that broke with the established mental model and I had to pause my work flow to figure out what was correct. A simple bolded emphasis of the phrase "both of which are acceptable" would increase skimmability and help transfer meaning from screen to brain even faster. 

## Relevant information

According to [this study from NNG](https://www.nngroup.com/articles/concise-scannable-and-objective-how-to-write-for-the-web/), "Only 3 [of 19] participants started reading text word by word," and "15 [almost 78%] participants always approached unfamiliar Web text by trying to scan it before reading it."

I'm on GitHub s o m e t i m e s. Emailing me is quicker and more reliable: uxeverything.designer@gmail.com
